releasenotes/notes/add-django-channels-support-44aefaa21961adff.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fixes django channels issue which resulted in server errors when requesting views
```

Comment:
Can we reword this a bit, it isn't really clear what is being fixed from this message.